packages/components/legacy_layout/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@buoysoftware/anchor-ui-legacy-layout",
+  "version": "0.30.0",
+  "main": "src/index.ts",
+  "license": "MIT",
+  "dependencies": {
+    "@buoysoftware/anchor-layout": "^0.30.0"
```

Comment:
I copied over this file from the other legacy packages. Not quite sure what the convention is regarding versioning for a new package. I assume we'll review/approve/merge this in and then run `yarn lerna version` or should I update the versions of all packages in this PR?

Reply:
Yeah, when you run yarn publish-build lerna will walk you through versioning.

Reply:
Sounds good, thanks!